plugin.video.carnet-meduza/resources/lib/carnet-meduza.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+import os, sys
+import urllib, urlparse, json
+import xbmcgui, xbmcplugin, xbmcaddon
+import requests
+from register import device
```

Comment:
That's a bit ugly, I missed the init code hanging out on the bottom. I would have expected that you would instantiate the class in default.py and have done all that in the init. Let me discuss with others about that. I also would have thought that you could accomplish this stuff using one of the modules that's already in the Kodi repo, but I might be wrong as I don't use them myself much. Let me check with more knowledgeable folks.